pact-tests/gas-goldens/resume.repl

```diff
@@ -0,0 +1,15 @@
+(module testmodule g
+  (defcap g () true)
+
+  (defschema schema result:string)
+
+  (defpact test ()
+    (step (let ((x:object{schema} {"result": "hi"})) (yield x)))
+    (step (resume {"result" := r} (+ r " Kadena")))))
+    
+
```

Comment:
If this is a repl file could, could you `(env-gas 0)` before running the `(continue-pact 1)`? It will get rid of the noise from the other things being executed.